docs/reference/commandline/save.md

```diff
@@ -53,6 +53,13 @@ $ docker save -o fedora-all.tar fedora
 $ docker save -o fedora-latest.tar fedora:latest
 ```
 
+### Save the image to a tar.gz file using gzip
+
+You can use gzip to save the image file and make the backup smaller.
+
+```bash
+docker save myimage:latest | gzip > myimage_latest.tar.gz
+```
```

Comment:
Can you add a blank line after this? (we've had some cases where a missing blank line lead to Markdown not rendering properly on docs.docker.com 😅 )

Reply:
Sure,It `s my mistake.😄